docker/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:jammy
+
+RUN apt-get update && \
+  DEBIAN_FRONTEND='noninteractive' \
+  DEBCONF_NONINTERACTIVE_SEEN='true' \
+  apt-get install --yes \
+    gfortran \
+    build-essential \
+    git \
+    curl \
+    libblas-dev \
+    liblapack-dev \
+    autoconf \
+    automake \
+    autoconf-archive \
+    libpython2.7-dev \
+    libpython2.7 \
+    python2.7 \
+    libtool \
+    libc6-dev \
+    mpich \
+    sudo \
+    nano
+
+RUN useradd \
+  --create-home \
+  lithomop_user && echo "lithomop_user:password" | chpasswd
+
+
+RUN sudo adduser lithomop_user sudo
+
+# Need to create this link so that pythia will compile
+RUN ln -s /usr/bin/python2.7 /usr/bin/python;
+
+WORKDIR /home/lithomop_user
+
+USER lithomop_user
+
+#apt will install pip3, so need to do this to install pip 2.7
+RUN curl -L -O "https://bootstrap.pypa.io/pip/2.7/get-pip.py"; echo "password" | sudo -S python2.7 get-pip.py; pip install setuptools; pip install ez_setup;
+
+
+ENV PETSC_DIR="/home/lithomop_user/petsc-2.3.3-p16"
+ENV PETSC_ARCH="/home/lithomop_user/petsc-2.3.3-p16/lib/linux-gnu-c-debug"
```

Comment:
`PETSC_ARCH` should either be unset ("prefix install") or just the identifier (as in `linux-gnu-c-debug`, or whatever you specify).